control-plane/subcommand/server-acl-init/command_test.go

```diff
@@ -2645,20 +2653,22 @@ func TestRun_ValidateLoginToken_SecondaryDatacenter(t *testing.T) {
 			})
 			require.NoError(t, err)
 
-			retry.Run(t, func(r *retry.R) {
-				tok, _, err := client.ACL().Login(&api.ACLLoginParams{
-					AuthMethod:  authMethodName,
-					BearerToken: jwtToken,
-					Meta:        map[string]string{},
-				}, &api.WriteOptions{})
-				require.NoError(r, err)
-
-				require.Equal(r, len(tok.Roles), len(c.Roles))
-				for _, role := range tok.Roles {
-					require.Contains(r, c.Roles, role.Name)
-				}
-				require.Equal(r, !c.GlobalToken, tok.Local)
-			})
+			if len(c.Roles) == 1 {
```

Comment:
I'm a bit confused by this logic. Why do we only do it if we have one role?

Reply:
Re-typing for completeness of other viewers. This was a remnant.